types/chroma-js/package.json

```diff
@@ -20,6 +20,20 @@
         {
             "name": "Charlie Zhuo",
             "githubUsername": "CharlieZhuo"
+        },
+        {
+            "name": "Justin Trenary",
+            "githubUsername": "trenaryja"
+        }
```

Comment:
This is a list of owners of the package which are in charge of reviewing it, not a contributors list. Are you intentionally adding yourself as an owner?

(You're a first time contributor; that combo is rare.)

Reply:
@jakebailey ah I didn't fully understand. I removed myself from the owner's list, I'm not sure if I should have the authority to review or not (I'm not even really sure who decides that). 

I would happily review changes in the future if that's helpful, but also don't want to overstep bounds with existing owners. I'm just a fan of the chroma-js library, I haven't contributed to it directly (yet).

Reply:
Typically other owners approve other owners, or maintainers approve if owners never appear and the user has a history of good contributions such that it'd speed up further changes.